### PR TITLE
Under Strict Confinement - AzIoTIdentity integration 

### PR DESCRIPTION
**Pull request by shiyi-peng**
Merging to feature/snap

### CODE REVIEW

**Review by Nox-MSFT — APPROVED** [review]

File: src/utils/eis_utils/src/eis_utils.c [hunk 1]
```diff
@@ -418,36 +432,197 @@ EISErr BuildSasCertConnectionString(
...
-// External Functions
-//
+/**
+ * @brief Writing EIS Identities information to a EIS named pipe
+ * @details Calls into the EIS Identity and get the identities response, saves to a named pipe
+ * @param[in] timeoutMS the timeoutMS in milliseconds for each call to EIS
+ * @param[in] filePath the path of the EIS response named pipe
+ * returns EISErr value
+ */
+// EISErr EISIdentitiesFileWriter(uint32_t timeoutMS, const char* filePath)
```
Comment (nihemstr) [inline src/utils/eis_utils/src/eis_utils.c:442]: why are we checking in large blocks of commented out code?

File: src/utils/eis_utils/src/eis_utils.c [hunk 2]
```diff
@@ -670,6 +835,523 @@ EISUtilityResult RequestConnectionStringFromEISWithExpiry(
...
+//  * @brief Creates a connection string using the provisioned data within EIS
+//  * @details Calls into the EIS Identity and Keyservice to create a SharedAccessSignature which is then used
+//  * to create the connection string, Caller is required to call free() to deallocate the connection string
+//  * @param[in] expirySecsSinceEpoch the expiration time in seconds since the epoch for the token in the connection string
+//  * @param[in] timeoutMS the timeoutMS in milliseconds for each call to EIS
+//  * @param[out] provisioningInfo the pointer to the struct which will be initialized with the information for creating a connection to IotHub using the EIS supported provisioning information
+//  * @returns on success a null terminated connection string, otherwise NULL
+//  */
+// EISUtilityResult RequestConnectionStringFromEISWithExpiry(
+//     const time_t expirySecsSinceEpoch, uint32_t timeoutMS, ADUC_ConnectionInfo* provisioningInfo)
```
Comment (nihemstr) [inline src/utils/eis_utils/src/eis_utils.c:882]: More commented out code...

**Review by nihemstr — CHANGES_REQUESTED** [review]
Need justification for the commented out sections

**Review by nihemstr — APPROVED** [review]
Approving changes to remove block

**Pull request merged**